### PR TITLE
Add custom week day format

### LINKED ISSUE #453: custom weekday format on week header
weekday format seems to be hardcoded

`<small>{moment().weekday(i).format('dd')}</small>`

it would be good to be able to set this up in the config

### PR DESCRIPTION
**Pull request by schaable**
Add a prop to customize the day format in the header.

Resolves #453  and #132

### CODE REVIEW

**Comment by ljharb** [comment]
Would you mind rebasing this on the command line so there's no merge commit?

**Comment by schaable** [comment]
Done!

**Comment by jseminck** [comment]
I would like the same functionality, but I think it's more consistent to provide a 1render()1 function rather than just specifying the format. The same idea applies for `renderDay` and `renderMonth` which already exist right now.

I have a branch in my fork that implements a `renderWeekHeaderDay()`  prop. Please have a look and let me know what you think: https://github.com/jseminck/react-dates/commit/494117bd86e0c0ad3812eea1d3a09afa4ae0d001

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ isDayHighlighted: PropTypes.func,
 // internationalization props
 displayFormat: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
 monthFormat: PropTypes.string,
+weekDayFormat: PropTypes.string,
```
Comment (ljharb) [inline README.md]: Is there a moment proptype we could use for "valid format"?

Comment (majapw) [reply README.md]: Technically I think any string is a valid format (you can include arbitrary strings by surrounding them with brackets)

**Pull request merged**